install_config/install/docker_registry.adoc

```diff
@@ -976,21 +970,38 @@ can be overridden using
 It will be set to `*true*` in future versions.
 <3> An expiration timeout for limits cached in the registry. The lower the
 value, the less time will it take for the limit changes to propagate to the
-registry. On the other hand, registry will query limits from the server more
+registry. However, registry will query limits from the server more
 frequently and, as a consequence, pushes will be slower.
 ====
 
 The link:https://docs.docker.com/registry/configuration/#cloudfront[*cloudfront*
 middleware extension] can be added to support AWS, CloudFront CDN storage
 provider.
 
+The *middleware* section may not be overridden using environment variables.
+There are a few exceptions, however. For example:
+====
+----
+middleware:
+  repository:
+    - name: openshift
+      options:
+        acceptschema2: false <1>
+        enforcequota: false <2>
+        projectcachettl: 1m <3>
+----
+<1> A boolean environment variable that allows to accept manifest schema v2 on manifest put requests.
```

Comment:
The reference line isn't an environment variable, it's just a config option that can be overridden by env var `REGISTRY_MIDDLEWARE_REPOSITORY_OPENSHIFT_ACCEPTSCHEMA2`. I'd definitely mention the variable name because it's hard to guess.

 The same for the other options. I'm re-pasting here the other env var names:

- `REGISTRY_MIDDLEWARE_REPOSITORY_OPENSHIFT_ENFORCEQUOTA`

- `REGISTRY_MIDDLEWARE_REPOSITORY_OPENSHIFT_PROJECTCACHETTL`